### PR TITLE
feat: output pnpm version with `info`/`version` command

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
No
**Summary**
Fixes https://github.com/webpack/webpack-cli/pull/3072/

Add `pnpm` information

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Comment by evenstensberg** [comment]
Is this still a draft?

**Comment by snitin315** [comment]
Yeah. I just need to fix the tests on CI, will do it today.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -82,6 +82,13 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: "yarn"
 
+      # Setup pnpm for tests
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:88]:
```suggestion
          version: 7
```

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -82,6 +82,13 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: "yarn"
 
+      # Setup pnpm for tests
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+      - name: Pnpm version
+        run: pnpm -v
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:90]:
```suggestion
```

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -82,6 +82,11 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: "yarn"
 
+      # Setup pnpm v7 for tests
```
Comment (snitin315) [inline .github/workflows/nodejs.yml:85]:
```suggestion
      # Setup pnpm v7 for tests which is compatible with Node.js >=14
```

**Comment by evenstensberg** [comment]
ok lmk

**Review by evenstensberg — APPROVED** [review]
lgtm

**Comment by evenstensberg** [comment]
Could you rebase to make commitlint happy?

**Comment by snitin315** [comment]
@evenstensberg I'll modify the commit message while merging the PR, it should be fine.

**Pull request merged**